tests/PHPUnit/Integration/ArchiveWithNoVisitsTest.php

```diff
@@ -78,6 +85,28 @@ public function test_getIdSitesToArchiveWhenNoVisits_CanBeUsedToTriggerArchiving
         $this->assertEquals($expectedMethodCalls, ArchiveWithNoVisitsTest_MockArchiver::$methodsCalled);
     }
 
+    public function test_PluginArchiver_CanBeUsedToTriggerArchiving_EvenIfSiteHasNoVisits()
+    {
+        PluginsArchiver::$archivers['VisitsSummary'] = 'Piwik\Tests\Integration\ArchiveWithNoVisitsTest_MockArchiver';
+
+        ArchiveWithNoVisitsTest_MockArchiver::$runWithoutVisits = true;
```

Comment:
let's add the similar test case with runWithoutVisits = false so the different behavior can be seen